satellite/satellitedb/offers.go

```diff
@@ -0,0 +1,157 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information
+
+package satellitedb
+
+import (
+	"context"
+	"database/sql"
+	"time"
+
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/satellite/marketing"
+	dbx "storj.io/storj/satellite/satellitedb/dbx"
+)
+
+type offers struct {
+	db *dbx.DB
+}
+
+// ListAll returns all offers from the db
+func (offers *offers) ListAll(ctx context.Context) ([]marketing.Offer, error) {
+	offersDbx, err := offers.db.All_Offer(ctx)
+	if err != nil {
+		return nil, marketing.OffersErr.Wrap(err)
+	}
+
+	return offersFromDbx(offersDbx)
+}
+
+// GetCurrent returns an offer that has not expired based on offer status
+func (offers *offers) GetCurrent(ctx context.Context, offerStatus marketing.OfferStatus) (*marketing.Offer, error) {
+	var statement string
+	const columns = "id, name, description, award_credit_in_cents, invitee_credit_in_cents, award_credit_duration_days, invitee_credit_duration_days, redeemable_cap, num_redeemed, expires_at, created_at, status"
+	statement = `
+		WITH o AS (
```

Comment:
another stylistic nit, I think `o` should just be `offer`

Reply:
`o` is temporary variable not a real table here. I feel it would be better to use a name that doesn't look like a table name here